index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <img src="/images/logo.svg" alt="Logo" />
+      <img src="/images/menu-icon.svg" alt="Menu Icon" />
+    </header>
+    <main class="hero-section">
+      <section>
+        <h1 class="hero-heading">I'm Zain Glad to see you!</h1>
```

Comment:
- [ ] Subtitle line should use hx

Reply:
I'm sorry, I didn't understand what hx is. Can you please explain?